packages/rs-drive/src/query/conditions.rs

```diff
@@ -1048,6 +1048,41 @@ impl<'a> WhereClause {
         Ok(query)
     }
 
+    /// Recursively find the property type for a given nested field name.
+    fn find_property_type(
+        document_type: &'a DocumentType,
+        field_name: &str,
```

Comment:
function is about `property`, but this arg is named `field_name`. What's the difference between property and field?

Reply:
Changed to property_name